test/experimental/test_pixel_mapping.py

```diff
@@ -0,0 +1,11 @@
+def test_pixel_mapping(file_camera):
+    from zivid.experimental.calibration import pixel_mapping
+    from zivid.settings import Settings
+
+    pixel_mapping_handle = pixel_mapping(
+        camera=file_camera, settings=Settings(acquisitions=[Settings.Acquisition()])
+    )
+    assert isinstance(pixel_mapping_handle.row_stride, int)
+    assert isinstance(pixel_mapping_handle.col_stride, int)
+    assert isinstance(pixel_mapping_handle.row_offset, float)
+    assert isinstance(pixel_mapping_handle.col_offset, float)
```

Comment:
We should also check the exact values that are used, IMO. To ensure that it is mapped correctly in the pybind wrapper.

Reply:
https://github.com/zivid/zivid-python/pull/251/commits/bb420327c73bdffc3c4cfcb514adadacd5137054